java/libphonenumber/src/com/google/i18n/phonenumbers/MultiFileMetadataSourceImpl.java

```diff
@@ -110,13 +110,18 @@ private boolean isNonGeographical(int countryCallingCode) {
   static <T> PhoneMetadata loadMetadataFromFile(
       T key, ConcurrentHashMap<T, PhoneMetadata> map, String filePrefix,
       MetadataLoader metadataLoader) {
+    // The size of the byte buffer used for deserializing the phone number metadata files for each
+    // region.
```

Comment:
Defined in MetadataManager, added comments (long-term I'll move all this logic there, doesn't make sense to have 3 extra files just for the regular loading).